src/components/dialogs/network-modification/composite-modification/composite-modification-edition-dialog.tsx

```diff
@@ -0,0 +1,169 @@
+import Box from '@mui/material/Box';
+import { setSelectionForCopy } from '../../../../redux/actions';
+import { useDispatch, useSelector } from 'react-redux';
+import { AppState } from '../../../../redux/reducer';
+
+const schema = yup.object().shape({
+    [FieldConstants.NAME]: yup.string().trim().required('nameEmpty'),
+});
+
+const emptyFormData = (name?: string) => ({
+    [FieldConstants.NAME]: name,
+});
+
+interface FormData {
+    [FieldConstants.NAME]: string;
+}
+
+interface CompositeModificationEditionDialogProps {
+    compositeModificationId: string;
+    open: boolean;
+    onClose: (event?: SyntheticEvent) => void;
+    titleId: string;
+    name: string;
+    broadcastChannel: BroadcastChannel;
+}
+
+export const CompositeModificationEditionDialog: FunctionComponent<CompositeModificationEditionDialogProps> = ({
+    compositeModificationId,
+    open,
+    onClose,
+    titleId,
+    name,
+    broadcastChannel,
+}: Readonly<CompositeModificationEditionDialogProps>) => {
+    const intl = useIntl();
+    const [languageLocal] = useParameterState(PARAM_LANGUAGE);
+    const [isFetching, setIsFetching] = useState(!!compositeModificationId);
+    const { snackError } = useSnackMessage();
+    const selectionForCopy = useSelector((state: AppState) => state.selectionForCopy);
+    const [modifications, setModifications] = useState<NetworkModificationMetadata[]>([]);
+    const dispatch = useDispatch();
+
+    const methods = useForm<FormData>({
+        defaultValues: emptyFormData(name),
+        resolver: yupResolver(schema),
+    });
+
+    const { computeLabel } = useModificationLabelComputer();
+    const getModificationLabel = (modif: NetworkModificationMetadata) => {
+        if (!modif) {
+            return null;
+        }
+        return intl.formatMessage(
+            { id: 'network_modifications.' + modif.type },
+            {
+                ...modif,
+                ...computeLabel(modif),
+            }
+        );
+    };
+
+    const renderNetworkModificationsList = () => {
+        return (
+            <>
+                {modifications && (
+                    <List sx={unscrollableDialogStyles.scrollableContent}>
+                        {modifications.map((modification: NetworkModificationMetadata) => (
+                            <>
+                                <ListItem key={modification.uuid}>
+                                    <Box>{getModificationLabel(modification)}</Box>
+                                </ListItem>
+                                <Divider component="li" />
+                            </>
+                        ))}
+                    </List>
+                )}
+            </>
+        );
+    };
+
+    useEffect(() => {
+        setIsFetching(true);
+        getCompositeModificationContent(compositeModificationId)
+            .then((response) => {
+                if (response) {
+                    setModifications(response);
```

Comment:
setCompositeModifications(response);
don't forget to update the  [modifications, setModifications] = useState
modifications is too generic

Reply:
I can change this to `setNetworkModifications` if you want, but those are not composite modifications.